specs/_features/whisk/beacon-chain.md

```diff
@@ -382,49 +367,30 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
 
 ### Deposits
 
+```python
+def get_initial_whisk_k(validator_index: ValidatorIndex, counter: int) -> BLSFieldElement:
+    return BLSFieldElement(bytes_to_bls_field(hash(uint_to_bytes(validator_index) + uint_to_bytes(uint64(counter)))))
```

Comment:
Future note: `bytes_to_bls_field()` will likely assert here since we are feeding it too many bytes (won't fit in the scalar field element modulus). But this is not introduced by this PR so let's ignore this for now.

Reply:
To be fixed when introducing tests :+1: